includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -81,7 +81,7 @@ public function capture_terminal_payment( $request ) {
 			}
 
 			// Set the payment method on the order.
-			$order->set_payment_method( WC_Payment_Gateway_WCPay::GATEWAY_ID );
+			$order->set_payment_method( $this->gateway::GATEWAY_ID );
```

Comment:
I updated this to use the current gateway's GATEWAY_ID instead of the default credit card.